### DISCUSSION TITLE
How to render templates as strings using lit-html?

### DISCUSSION THREAD

**Discussion by idranme** [lit/lit#4392] [Q&A] [answered]
I only want strings, not rendered to DOM.

**Comment by augustjk** [comment, marked as answer]
https://github.com/lit/lit/tree/main/packages/labs/ssr is meant to produce HTML string from lit-html templates. The generated HTML includes marker comments to serve as hydration points, but also has a server-only `html` tag function that can be used to signal that it's not meant to hydrate and not produce those marker comments.
